psh/process.py

```diff
@@ -1164,6 +1164,16 @@ def __to_str(self):
 
                 command += arg
 
+        if self.__stdout_target is STDERR:
+            command += " >&2"
+        elif isinstance(self.__stdout_target, File):
+            command += " > {0}".format(self.__stdout_target.path)
```

Comment:
File paths must be quoted out the same way as command arguments.

I suggest to add a module function

```python
def _arg_to_str(arg)
    """Returns a string representation (possibly quoted) of a command argument as it would be used in shell scripts.

    .. note::
        Very lazy formatting. Not suitable for automatic generation of shell scripts.
    """
```
and use it for argument and file path convertion to strings.